source/pathSatisfies.js

```diff
@@ -23,6 +23,10 @@ import path from './path';
  *      R.pathSatisfies(R.is(Object), [], {x: {y: 2}}); //=> true
  */
 var pathSatisfies = _curry3(function pathSatisfies(pred, propPath, obj) {
-  return pred(path(propPath, obj));
+  var result = false;
+  try {
+    result = pred(path(propPath, obj));
```

Comment:
IMO. There is the best way to using `pathOr` instead of try...catch

Reply:
Yes, `pathOrSatisfies` does not swallow an error throw from predication:

```js
var pathOrSatisfies = _curry3(function pathOrSatisfies(defaultValue, pred, propPath, obj) {
  result = pred(pathOr(defaultValue, propPath, obj));
  return result;
});
```
@CrossEye